sound/soc/sof/core.c

```diff
@@ -382,14 +402,24 @@ static int sof_probe(struct platform_device *pdev)
 	else
 		sdev->boot_timeout = plat_data->desc->boot_timeout;
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_PROBE_WORK_QUEUE)
+	schedule_work(&sdev->probe_work);
+	return 0;
+#else
 	return sof_probe_continue(sdev);
+#endif
+
```

Comment:
looks like one empty line too many to me

Reply:
it's usual to have one line after an endif. Checkpatch doesn't complain here :-)